adafruit_irremote.py

```diff
@@ -50,11 +50,8 @@
   https://github.com/adafruit/circuitpython/releases
 
 """
-
-# Pretend self matter because we may add object level config later.
-# pylint: disable=no-self-use
```

Comment:
Oh I just noticed this line! That explains why `pylint` worked before. :upside_down_face: Oops.

Well, disabling it in a more targeting way (as I'm doing here) is probably "better" anyway.